code/datums/autolathe/tools_1.dm

```diff
@@ -13,11 +13,11 @@
 	build_path = /obj/item/device/multitool
 	materials = list(DEFAULT_WALL_MATERIAL = 62, "glass" = 25)
 
-/datum/design/item/autolathe/tools/t_scanner
+/*/datum/design/item/autolathe/tools/t_scanner
 	name = "T-ray scanner"
 	id = "t_scanner"
 	build_path = /obj/item/device/t_scanner
-	materials = list(DEFAULT_WALL_MATERIAL = 187)
+	materials = list(DEFAULT_WALL_MATERIAL = 187) already exists in normal designs */
```

Comment:
If it already exists, then it's probably not needed in this code file?

Reply:
i'm seeing if i'll really remove it, since the designs from protolathe have no category, so it woulnd't get sorted anywhere, the only reason i'd make it remove is because it gives error on a unit test that checks if it has 2 or more recipes with same build_path, which for this is irrelevant